docs/checks.md

```diff
@@ -601,3 +601,13 @@ possible.
 **Remediation steps**
 - Fix the vulnerabilities. The details of each vulnerability can be found on <https://osv.dev>.
 
+## Webhooks 
+
+Risk: `High` (service possibly accessible to third parties)
```

Comment:
I think we could even set it to critical, because the endpoint can be reached by arbitrary users on the Internet

Reply:
done